CONTRIBUTING.md

```diff
@@ -32,9 +32,8 @@ Every Issue and PR should be tagged with one or more of the following labels to
   - Project Name = Workstream Name.
 - Issues must be correctly set with the right labels and Milestone by WSL.
   - Relevant Milestone for the upcoming release (M1 - M6).
-- Workstream Leads have the sole responsibility to assign issues to individuals to address them.
-- **self assignment** are NOT recommended. Contributors may ask WSL (via comment or others) to get the issue assigned to them if they would like to work on it.
-  - This is to make sure items are worked based on priorities and right reasoning.
+- Contributors can assign the issues to themselves if they would like to work on them
```

Comment:
Can we change to "Contributors can assign an issue to themselves if they would like to work on it"
Personally have an issue with "the issues"

Reply:
fixed